File: modified_scripts/c/is_palindrome/gemma_2b.c

int is_palindrome(char *str)
{
    int n = strlen(str);
    int i;
    int j;

    for (i = 0; i < n / 2; i++)
    {
        if (str[i] != str[n - i - 1])
        {
            return 0;
        }
    }

    return 1;
}

File: original_scripts/python/is_palindrome/gemma_2b.py
```python
def is_palindrome(word):
    """
    Checks if a word is a palindrome.

    A palindrome is a word that reads the same backward as forward.

    Args:
        word (str): The word to check.

    Returns:
        bool: True if the word is a palindrome, False otherwise.
    """

    return word == word[::-1]
```